style.css

```diff
@@ -0,0 +1,228 @@
+.mobile-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#shape {
+  position: relative;
+  width: 74px;
+  height: 74px;
+  left: 28px;
+  top: -9px;
+}
+
+#shape1 {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+#logo {
+  position: absolute;
+  width: 74px;
+  height: 74px;
+  left: 34px;
+  top: 0;
+  background: url(images/Shape.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-logo {
+  position: absolute;
+  width: 72px;
+  height: 74px;
+  left: 0;
+  top: 0;
+  background: url(images/Shape1.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-menu {
+  display: flex;
+  height: 78px;
+  align-items: center;
+  width: 20px;
+  background: url(images/ic_menu.png);
+  background-repeat: no-repeat;
+}
+
+.main-header {
+  display: none;
+}
+
+.projects-block h2 {
+  font-size: 4em;
+  font-family: Playfair Display, serif;
+  margin-right: 20px;
+}
+
+#primary-text h1 {
+  font-size: 50px;
+  font-family: Playfair Display, serif;
+  padding: 0 20px;
+  color: #1a2236;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 100%;
```

Comment:
- [ ] Please keep in mind that the `line-height` property only requires a `decimal` number without any unit. Because of using the `line-height` incorrectly your design has a small fault.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-29 16-08-05](https://user-images.githubusercontent.com/66476329/235462648-09f4c2be-3e16-470f-9ccd-b122de850c30.png)

</td>
<td>

![Screenshot from 2023-05-01 19-07-55](https://user-images.githubusercontent.com/66476329/235462682-5224b7f2-b987-45a7-9d67-027e0d585e4a.png)

</td>
</tr>
</table>

Reply:
Hello @skyv26 please clarify more on this, because the line-height property is from the Figma design, and the 

> line-height property only requires a decimal number without any unit. 

The line-height documentation does not say so. 
<img width="793" alt="Screenshot 2023-05-01 at 4 01 17 PM" src="https://user-images.githubusercontent.com/10832117/235484902-4ebefbb7-4b45-4b54-a944-df37f117d081.png">

Reply:
@ctokoli, Thank you ❤️  for asking me the question ❓  and I am happy to answer your query. I would like to recommend using the `decimal` values because the default value of `line-height` is `1.2`. It will be easier for you to adjust the `line-height` between two lines. Overall, using unitless values is a best practice for *line-height* and can help improve the overall readability and accessibility of your web content.

## How does it work?
- Using unitless values in the line-height CSS property is recommended, as it makes the line height relative to the font size.
- For example, setting *line-height: 1.5;* means that the line height will be 1.5 times the font size, regardless of the unit used for the font size (e.g., px, em, rem, etc.). This can help ensure consistent line spacing across different font sizes, and can also make it easier to adjust line height when changing font size.
- Using unit-based values, such as *line-height: 24px;* can be problematic if the font size changes, as the line height will not adjust automatically.